src/FffGcodeWriter.cpp

```diff
@@ -535,52 +521,24 @@ void FffGcodeWriter::processLayer(SliceDataStorage& storage, int layer_nr, unsig
 
         if (layer_nr >= 0)
         {
-            std::vector<unsigned int>& mesh_order = mesh_order_per_extruder[extruder_nr];
-            unsigned int mesh_order_idx_starting_mesh = 0;
-            { // calculate mesh_order_idx_starting_mesh
-                Point layer_start_position = last_position_planned;
-                if (storage.getSettingBoolean("start_layers_at_same_position"))
```

Comment:
Exactly, that was the frontend commit I forgot to push ;)
This is *the* negative side effect of this PR.

Good review job!